code/src/main/java/amos/specitemdatabase/controller/Controller.java

```diff
@@ -52,4 +54,16 @@ public ResponseEntity<String> uploadDocument (@PathVariable(name="filename") Str
         System.out.println("Upload Successful!");
         return new ResponseEntity<>("Upload Successful!", HttpStatus.CREATED);
     }
+
```

Comment:
Can you add this code to this line **_'@CrossOrigin(origins = "http://localhost:3000")'_**, it enables to take the response from  port 3000

Reply:
Actually, this isn't part of my issue anymore. Anyways, I can do it.

Reply:
Ah okay, anyways thanks!